src/compiler/commandLineParser.ts

```diff
@@ -2641,6 +2642,18 @@ export function convertToTSConfig(configParseResult: ParsedCommandLine, configFi
     return config;
 }
 
+function optionDependsOn(option: string, dependsOn: Set<string>): boolean {
+    const seen = new Map<string, true>();

```

Comment:
Should this just be a Set?

(I'm guessing addToSeen is leftover from pre es6)

Reply:
I guess it has some handy behavior)

Reply:
This has bugged me for a while, maybe I should update `addToSeen`